portal-api/src/v1/api.js

```diff
@@ -662,6 +662,26 @@ const getAllPortalFacilityAmendments = async (statuses) => {
   }
 };
 
+/**
+ * Gets portal facility amendments on the deal filtered by status
```

Comment:
```suggestion
 * Gets the latest portal facility amendment on the facility filtered by status
```

For my understanding this is only one amendment that you'll get. Right?

Reply:
sorry this was documentation copied.  I have corrected it